### PR TITLE
Added a task pangram

### PR DESCRIPTION
**Pull request by annaoliinyk**
https://exercism.org/tracks/python/exercises/pangram

Instructions
Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan gramma, "every letter") is a sentence using every letter of the alphabet at least once. The best known English pangram is:

The quick brown fox jumps over the lazy dog.

The alphabet used consists of letters a to z, inclusive, and is case insensitive.

### CODE REVIEW

File: pangram/pangram_file.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Task from https://exercism.org/tracks/python/exercises/pangram
+
+def is_pangram(sentence):
+    sentence = sentence.lower()
+    alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u',
```
Comment (AnaKovaGH) [inline pangram/pangram_file.py]: It should be constant

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was requested above

File: pangram/pangram_file.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+def is_pangram(sentence):
+    sentence = sentence.lower()
+    for letter in alphabet:
+        if letter not in sentence:
+            return False
+    return True
+
+
+def main():
+    print(is_pangram('a quick movement of the enemy will jeopardize five gunboats'))
```
Comment (AnaKovaGH) [inline pangram/pangram_file.py]: Please change print to logger output

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change print to logger

**Comment by annaoliinyk** [comment]
Ready for review

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**